app/backend/.env.local

```diff
@@ -4,14 +4,20 @@ API_PORT=3005
 DB_URI=postgresql://postgres:postgres@localhost:5432/postgres
 DB_FRESH_START=true
 
-ETHEREUM_MONITOR_MODULE_ENABLED=false
-ETHEREUM_RPC_ENDPOINT=
-ETHEREUM_MONITOR_TASK_INTERVAL_MS=10000
-ETHEREUM_LOGS_START_BLOCK=19119000
-ETHEREUM_LOGS_MAX_BLOCKS_PER_LOG_FETCH=5000
-
 PRICING_MODULE_ENABLED=true
 PRICING_COINCAP_BASE_URL=https://api.coincap.io/v2
 PRICING_COINCAP_API_KEY=
 PRICING_MINUTE_RATE_LIMIT=500
 
+ETHEREUM_RPC_ENDPOINT=
+OPTIMISM_RPC_ENDPOINT=
+
+ETHEREUM_MONITOR_MODULE_ENABLED=false
```

Comment:
Is Ethereum module not implemented yet or why do you prefer to disable it by default?